xml_converter/generators/code_generator.py

```diff
@@ -187,6 +191,16 @@
                     type: string
             uses_file_path:
                 type: boolean
+            xml_parent_export:
+                type: array
+                items:
+                    type: string
+                    pattern: "^[A-Za-z ]+$"
```

Comment:
https://github.com/AsherGlick/Burrito/pull/134#discussion_r1174662042

Reply:
Just remove this regex like in compound.